wp-content/themes/oceanwp-child/style.css

```
@@ -59,4 +62,11 @@ button {
     color: #ffffff;
     white-space: nowrap;
     border: 0;
+}
+
+form textarea#comment.textarea-comment{
```

General question - where is this being used? I couldn't find a single `<textarea>` in the code. Nor could I find where `.textarea-comment` is being used.

CSS question - Wouldn't this work as well:
```suggestion
#comment{
```
An id is supposed to be unique in the page.
If it is not, it is a problem.